requirements.txt

```diff
@@ -2,7 +2,7 @@ gemd==1.16.0
 requests==2.31.0
 pyjwt==2.4.0
 arrow==0.15.4
-sphinx==4.3.0
+sphinx==5.0.0
```

Comment:
The GitHub Actions build and deploy actions require Sphinx 5.0. The full upgrade will be in the 3.0 release.